oggsound/src/OgreOggSoundManager.cpp

```diff
@@ -2501,26 +2499,32 @@ namespace OgreOggSound
 	/*/////////////////////////////////////////////////////////////////*/
 	int OgreOggSoundManager::_createSourcePool()
 	{
-		ALuint source;
-		unsigned int numSources = 0;
+		ALuint sources[mMaxSources];
+
+		mSources = new ALuint[mMaxSources];
```

Comment:
```suggestion
		mSources.resize(mMaxSources);
```
otherwise the memory is leaked